apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -11,46 +11,110 @@
-        </v-list-item-content>
-      </v-list-item>
-      <v-list-item to="/compare" title="Compare evaluations' controls">
-        <v-list-item-avatar>
-          <v-icon small>mdi-triangle-outline</v-icon>
-        </v-list-item-avatar>
-
-        <v-list-item-content>
-          <v-list-item-title>Comparison</v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
+      <div v-if="visible_evaluation_files.length > 0">
+        <v-list-item to="/results" title="View results">
+          <v-list-item-avatar>
+            <v-icon small>mdi-television-guide</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              Results View
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
+      <div v-else>
+        <v-list-item title="View results">
+          <v-list-item-avatar>
+            <v-icon small>mdi-television-guide</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              Results View (why is link deactivated?)
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
+      <div v-if="visible_evaluation_files.length > 0">
+        <v-list-item to="/compare" title="Compare results">
+          <v-list-item-avatar>
+            <v-icon small>mdi-triangle-outline</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              Results Comparison View
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
+      <div v-else>
+        <v-list-item title="Compare results">
+          <v-list-item-avatar>
+            <v-icon small>mdi-triangle-outline</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              Results Comparison View (why is link deactivated?)
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
+      <div v-if="visible_profile_files.length > 0">
+        <v-list-item to="/profiles" title="View profiles">
+          <v-list-item-avatar>
+            <v-icon small>mdi-television-guide</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              Profiles View
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
+      <div v-else>
+        <v-list-item title="View profiles">
+          <v-list-item-avatar>
+            <v-icon small>mdi-television-guide</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              Profiles View (why is link deactivated?)
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
```

Comment:
This all still needs to be removed correct?

Reply:
Yes, the new UI design removes these completely.